src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Dispatch, SetStateAction, useEffect } from 'react';
+import cn from 'classnames';
+import { ErrorType } from '../types/ErrorType';
+
+type Props = {
+  errorMessage: ErrorType;
+  setErrorMessage: Dispatch<SetStateAction<ErrorType>>;
```

Comment:
the same but easier don't u think?
```suggestion
  setErrorMessage: (error: ErrorType) => void;
```